composeApp/src/commonMain/kotlin/com/github/familyvault/services/FileCabinetService.kt

```diff
@@ -36,71 +46,270 @@ class FileCabinetService(
             managers = managers,
             tag = AppConfig.FILE_CABINET_THREAD_TAG,
             type = FileCabinetThreadType.IMAGES.toString(),
-            name = familyGroupName,
-            referenceStoreId = storeId,
+            name = "$familyGroupName$IMAGES_SUFFIX",
+            referenceStoreId = imagesStoreId,
+            threadInitialCreators = emptyList()
+        )
+
+        val documentsStoreId = privMxClient.createStore(
+            contextId,
+            users,
+            managers,
+            StoreType.FILE_CABINET_DOCUMENTS.toString()
+        )
+        
+        privMxClient.createThread(
+            contextId = contextId,
+            users = users,
+            managers = managers,
+            tag = AppConfig.FILE_CABINET_THREAD_TAG,
+            type = FileCabinetThreadType.DOCUMENTS.toString(),
+            name = "$familyGroupName$DOCUMENTS_SUFFIX",
+            referenceStoreId = documentsStoreId,
             threadInitialCreators = emptyList()
```

Comment:
Nie powinniśmy robić Thread, ale niestety wcześniej już ten sposób przeszedł. Jest na to jakiś task ale niech to już zostanie tak jak jest.